src/Platforms.jl

```diff
@@ -146,3 +146,4 @@ const ARCHITECTURE_FLAGS = Dict(
     ),
 )
 march(p::AbstractPlatform; default=nothing) = get(tags(p), "march", default)
+sanitizer(p::AbstractPlatform; default=nothing) = get(tags(p), "sanitizer", default)
```

Comment:
Do we want to call this `sanitizer` or `sanitize`? The latter would match better with `-fsanitize`.

Reply:
I like `sanitize`